<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementDialog</class>
 <widget class="QDialog" name="ElementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>948</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Atom Number:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblAtomicNumber">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Symbol:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblSymbol">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Summary:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lblSummary">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Atomic Mass:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblAtomicMass">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Atomic Radius:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="lblAtomicRadius">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Boiling Point:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="lblBoil">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Period:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="lblPeriod">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Melting Point:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="lblMelt">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Group:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="lblGroup">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Category:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="14" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Metallic Property:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="15" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Ionization Energy:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="16" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Block:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Electron Affinity:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="18" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Phase:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="19" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Density:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="20" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Discovery Year:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="21" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Discovered By:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Electron Configuration:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="lblElectronConfiguration">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Electronegativity:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="lblElectronegativity">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="lblCategory">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Oxidation States:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="lblOxidationStates">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="lblMetallicProperty">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="lblIonizationEnergy">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="lblBlock">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="lblElectronAffinity">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="lblPhase">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="lblDensity">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="lblDiscoveryYear">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLabel" name="lblDiscoveredBy">
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
